package.json

```diff
@@ -22,6 +22,10 @@
   },
   "homepage": "https://github.com/digitalbazaar/data-integrity-test-suite-assertion#readme",
   "devDependencies": {
+    "@digitalbazaar/bbs-2023-cryptosuite": "^1.2.0",
+    "@digitalbazaar/ecdsa-rdfc-2019-cryptosuite": "^1.0.1",
+    "@digitalbazaar/ecdsa-sd-2023-cryptosuite": "^3.2.1",
+    "@digitalbazaar/eddsa-rdfc-2022-cryptosuite": "^1.0.1",
```

Comment:
Is there a reason these were moved to `devDependencies`?

Reply:
Then why are they still in the dependency list here?

Reply:
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#devdependencies

> If someone is planning on downloading and using your module in their program, then they probably don't want or need to download and build the external test or documentation framework that you use.
> 
> In this case, it's best to map these additional items in a devDependencies object.
> 

The devDeps in this case are for the test project.

Reply:
For the tests that test this test project? 😜